### PR TITLE
WIP: examples folder refactoring

### PR DESCRIPTION
**Pull request by giacomocerquone**
---
name: :book: New/Updated Documentation Content
about: Adding a new docs page, or updating content in an existing docs page
---

## PR Type

**Does this PR add a _new_ page, or update an _existing_ page?**

## Checklist

- [x] Is there an existing issue for this PR?
  - #3917 
- [x] Have the files been linted and formatted?

## What docs page is being added or updated?

- **Section**: Introduction
- **Page**: Examples

### What updates should be made to the page?
Linking updated redux examples

### Do these updates change any of the assumptions or target audience? If so, how do they change?
No

### CODE REVIEW

**Comment by markerikson** [comment]
Ho-ly smoke, that's a big PR :)  

(I know, I know, most of it is `package-lock.json` files being removed).

First observation: since there may be external links that still point to those folders, can we replace the dead examples with small `README.md` files that have specific suggestions on what other examples to look at instead? ie, `examples/todomvc/README.md` might say "Go look at the new Redux Fundamentals example todo app -> here".

File: docs/introduction/Examples.md [hunk 1]
```diff
@@ -160,24 +123,6 @@ This example demonstrates rendering a deeply nested tree view and representing i
 
 This example includes tests.
 
-## Async
```
Comment (markerikson) [inline docs/introduction/Examples.md:163]: Let's replace the "Async" entry here with the "Redux Essentials" example app example instead, including description and final source CodeSandbox.

**Comment by giacomocerquone** [comment]
@markerikson wait wait, I did nothing else but removing the folders we said to remove and replacing counter with "npx create-react-app --template=redux"

**Review by markerikson — CHANGES_REQUESTED** [review]

**Comment by markerikson** [comment]
I'm fine with the one PR - just saying that `+12.2K -66K` is a pretty big diff :)

**Comment by giacomocerquone** [comment]
YOLO 😄

File: docs/introduction/Examples.md (hunk 1 above)
Comment (giacomocerquone) [reply docs/introduction/Examples.md:163]: essentials too?

**Review by markerikson — CHANGES_REQUESTED** [review]

**Comment by timdorr** [comment]
Please don't switch to Yarn. We use npm for this repo.

**Comment by markerikson** [comment]
@timdorr good catch, I missed that.

Although I think I may have used Yarn for the "Essentials" tutorial repo?  At least the README file ended up saying `yarn build`, etc, although I didn't include a lockfile in there.

**Comment by timdorr** [comment]
I don't really care much about the tooling in other repos. It's just we should use one package manager per repo, so that tooling usage will be consistent. And so the install scripts for the examples have the right lock file format available (even though npm 7 can read yarn.lock now)

**Comment by giacomocerquone** [comment]
Let me know if I can improve this pr anytime :)
Just ping!

**Review by juzhiyuan — COMMENTED** [review]
TL;DR we’d better split this PR into more smaller PRs.

**Comment by mindtraveller** [comment]
we've got a solution for such PRs like this. Just check it out here: https://viezly.com/change_requests/4135. Guess that way it'd be possible to review such refactoring.
